pkg/docker/docker.go

```diff
@@ -542,6 +666,9 @@ func (d *stiDocker) GetScriptsURL(image string) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	if imageMetadata == nil {
```

Comment:
also, I do remember later on adding a nil check in the private getScriptsURL (for path other than CheckAndPull).  So yeah, this is no longer needed.